Transbank/Transbank.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageId>TransbankSDK</PackageId>
     <Description>This is the .Net SDK for Transbank Integration.</Description>
     <PackageLicense>https://raw.githubusercontent.com/TransbankDevelopers/transbank-sdk-dotnet/master/LICENSE</PackageLicense>
-    <PackageProjectUrl>http://www.transbankdevelopers.cl/</PackageProjectUrl>
+    <PackageIconUrl>https://www.transbankdevelopers.cl//favicon.ico</PackageIconUrl>
```

Comment:
¿Tiene doble // por alguna razón? ¿Porque se elimina el PackageProjectUrl?

Reply:
Hey @gonzunigad good catch. I did replace the wrong tag.